sdk/remoterendering/azure-mixedreality-remoterendering/samples/remote_rendering_client_sample.py

```diff
@@ -108,14 +111,14 @@ def perform_asset_conversion():
         # input_settings blob_prefix more details at:
         # https://docs.microsoft.com/en-us/azure/remote-rendering/resources/troubleshoot#conversion-file-download-errors
         input_settings = AssetConversionInputSettings(
-            storage_container_uri=storage_container_uri,
+            storage_container_uri=str(storage_container_uri),
```

Comment:
if we use os.environ["ENVIRONMENT_VAR_NAME"] then python will raise if the env var is not set. We can also type the variable as string so that we don't need to call str() here:

storage_container_url: str = os.environ["ARR_STORAGE_CONTAINER_URI"]

Reply:
Added the type hint and removed the str() call.